src/services/publicCollectionDownloadManager.ts

```diff
@@ -37,9 +37,16 @@ class PublicCollectionDownloadManager {
                     const cacheResp: Response = await thumbnailCache?.match(
                         file.id.toString()
                     );
-                    console.log(cacheResp);
                     if (cacheResp) {
-                        return URL.createObjectURL(await cacheResp.blob());
+                        console.log('cache hit', file.id);
+                        const cacheBlob = await cacheResp.blob();
+                        console.log(
```

Comment:
Do we want to log the entire object or just object size/identifier will be good enough.
Also, it can potentially be just tested using cloudflare pages/pr specific deployment?